frontend/src/components/people/detail/PeopleDetailSideTeam.vue

```diff
@@ -50,7 +50,7 @@
         @close="dialog.close"
         @confirm="deleteTeam"
       >
-        <div v-html="t('people.deleteTeamText', { name: team.name, number: team.users.length })" />
```

Comment:
here I use v-html, because there is a <b></b> tag that highlights the name, it would be necessary to return to v-html

Reply:
Oke, reverse this script to div again